common/test/acceptance/pages/lms/teams.py

```diff
@@ -281,6 +286,20 @@ def cancel_team(self):
         self.q(css='.create-team .action-cancel').first.click()
         self.wait_for_ajax()
 
+    @property
+    def delete_team_button(self):
+        """Returns the 'delete team' button."""
+        return self.q(css='.action-delete').first
+
+    def confirm_delete_team_dialog(self):
+        """Click 'delete' on the warning dialog."""
+        confirm_prompt(self, require_notification=False)
+        self.wait_for_ajax()
+
+    def cancel_delete_team_dialog(self):
```

Comment:
Nit: I probably would call ``confirm_prompt`` directly from the test and not have this helper method. It seems heavyweight to have to add such a method for every modal that we might show on a page.